torch_geometric/nn/glob/__init__.py

```diff
@@ -35,3 +27,48 @@
 class GlobalAttention(AttentionalAggregation):
     def __call__(self, x, batch=None, size=None):
         return super().__call__(x, batch, dim_size=size)
+
+
+sum_aggr = SumAggregation()
+
+
+@deprecated(
```

Comment:
Let's not deprecate them, they are used quite heavily. Maybe we can just move`glob/glob.py` to `pool/glob.py` instead. Here, we can then just deprecate its usage from the `nn.glob` package.

Reply:
Makes sense will update